src/migration/1710235892939-MarketingCta.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class MarketingCta1710235892939 implements MigrationInterface {
+    name = 'MarketingCta1710235892939'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`CREATE TABLE "marketing_cta" ("campaignId" text NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "variant" text NOT NULL, "flags" jsonb NOT NULL DEFAULT '{}', CONSTRAINT "PK_28d00a5e83072d15724c50f8520" PRIMARY KEY ("campaignId"))`);
```

Comment:
Missing Replica identity full since it will be used with workers in the future.

Reply:
Does that have to be added manually?

Reply:
Yes, there are examples in codebase.

Reply:
On `marketing_cta`, `user_marketing_cta` or both?

Reply:
We can add both.

Reply:
done